packages/react-cards/src/demo/ColorStyles.scss

```diff
@@ -0,0 +1,26 @@
+@import '~office-ui-fabric-react/dist/sass/References';
+
+:global {
+  .hljs {
```

Comment:
Would be nice if we could share this and other demo code between packages somehow (probably out of scope for this change)

Reply:
That would be nice! But yeah, I'd like to keep that change separate from this PR.